src/mlpack/tests/lmnn_test.cpp

```diff
@@ -293,61 +290,61 @@ BOOST_AUTO_TEST_CASE(LMNNSeparableEvaluateWithGradientTest)
 
   double objective = lmnnfn.EvaluateWithGradient(coordinates, 0, gradient, 1);
 
-  BOOST_REQUIRE_CLOSE(objective, 1.576, 1e-5);
+  REQUIRE(objective == Approx( 1.576).epsilon(1e-7));
 
-  BOOST_REQUIRE_CLOSE(gradient(0, 0), -0.048, 1e-5);
-  BOOST_REQUIRE_CLOSE(gradient(0, 1), 0.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(gradient(1, 0), 0.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(gradient(1, 1), 2.0, 1e-5);
+  REQUIRE(gradient(0, 0) == Approx( -0.048).epsilon(1e-7));
+  REQUIRE(gradient(0, 1) == Approx( 0.0).epsilon(1e-7));
+  REQUIRE(gradient(1, 0) == Approx( 0.0).epsilon(1e-7));
+  REQUIRE(gradient(1, 1) == Approx( 2.0).epsilon(1e-7));
 
   objective = lmnnfn.EvaluateWithGradient(coordinates, 1, gradient, 1);
 
-  BOOST_REQUIRE_CLOSE(objective, 1.576, 1e-5);
+  REQUIRE(objective == Approx( 1.576).epsilon(1e-7));
 
-  BOOST_REQUIRE_CLOSE(gradient(0, 0), -0.048, 1e-5);
-  BOOST_REQUIRE_CLOSE(gradient(0, 1), 0.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(gradient(1, 0), 0.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(gradient(1, 1), 2.0, 1e-5);
+  REQUIRE(gradient(0, 0) == Approx( -0.048).epsilon(1e-7));
+  REQUIRE(gradient(0, 1) == Approx( 0.0).epsilon(1e-7));
+  REQUIRE(gradient(1, 0) == Approx( 0.0).epsilon(1e-7));
+  REQUIRE(gradient(1, 1) == Approx( 2.0).epsilon(1e-7));
```

Comment:
```suggestion
  REQUIRE(gradient(0, 0) == Approx(-0.048).epsilon(1e-7));
  REQUIRE(gradient(0, 1) == Approx(0.0).epsilon(1e-7));
  REQUIRE(gradient(1, 0) == Approx(0.0).epsilon(1e-7));
  REQUIRE(gradient(1, 1) == Approx(2.0).epsilon(1e-7));
```

Remove the extra space.